src/poisson_fft.f90

```diff
@@ -110,89 +110,89 @@ subroutine waves_set(self, xdirps, ydirps, zdirps, sz)
 
       ! periodic-x
       do i = 1, nx/2 + 1
-         w = 2*pi*(i - 1)/nx
-         wp = xdirps%stagder_v2p%a*2*xdirps%d*sin(0.5_dp*w) &
-              + xdirps%stagder_v2p%b*2*xdirps%d*sin(1.5_dp*w)
-         wp = wp/(1._dp + 2*xdirps%stagder_v2p%alpha*cos(w))
-
-         xkx(i) = cmplx(1._dp, 1._dp, kind=dp)*(nx*wp/xdirps%L)
-         exs(i) = cmplx(1._dp, 1._dp, kind=dp)*(nx*w/xdirps%L)
-         xk2(i) = cmplx(1._dp, 1._dp, kind=dp)*(nx*wp/xdirps%L)**2
+        w = 2*pi*(i - 1)/nx
+        wp = xdirps%stagder_v2p%a*2*xdirps%d*sin(0.5_dp*w) &
+          + xdirps%stagder_v2p%b*2*xdirps%d*sin(1.5_dp*w)
+        wp = wp/(1._dp + 2*xdirps%stagder_v2p%alpha*cos(w))
+
+        xkx(i) = cmplx(1._dp, 1._dp, kind=dp)*(nx*wp/xdirps%L)
+        exs(i) = cmplx(1._dp, 1._dp, kind=dp)*(nx*w/xdirps%L)
+        xk2(i) = cmplx(1._dp, 1._dp, kind=dp)*(nx*wp/xdirps%L)**2
       end do
       do i = nx/2 + 2, nx
-         xkx(i) = xkx(nx - i + 2)
-         exs(i) = exs(nx - i + 2)
-         xk2(i) = xk2(nx - i + 2)
+        xkx(i) = xkx(nx - i + 2)
+        exs(i) = exs(nx - i + 2)
+        xk2(i) = xk2(nx - i + 2)
       end do
 
       ! periodic-y
       do i = 1, ny/2 + 1
-         w = 2*pi*(i - 1)/ny
-         wp = ydirps%stagder_v2p%a*2*ydirps%d*sin(0.5_dp*w) &
-              + ydirps%stagder_v2p%b*2*ydirps%d*sin(1.5_dp*w)
-         wp = wp/(1._dp + 2*ydirps%stagder_v2p%alpha*cos(w))
-
-         yky(i) = cmplx(1._dp, 1._dp, kind=dp)*(ny*wp/ydirps%L)
-         eys(i) = cmplx(1._dp, 1._dp, kind=dp)*(ny*w/ydirps%L)
-         yk2(i) = cmplx(1._dp, 1._dp, kind=dp)*(ny*wp/ydirps%L)**2
+        w = 2*pi*(i - 1)/ny
+        wp = ydirps%stagder_v2p%a*2*ydirps%d*sin(0.5_dp*w) &
+          + ydirps%stagder_v2p%b*2*ydirps%d*sin(1.5_dp*w)
+        wp = wp/(1._dp + 2*ydirps%stagder_v2p%alpha*cos(w))
+
+        yky(i) = cmplx(1._dp, 1._dp, kind=dp)*(ny*wp/ydirps%L)
+        eys(i) = cmplx(1._dp, 1._dp, kind=dp)*(ny*w/ydirps%L)
+        yk2(i) = cmplx(1._dp, 1._dp, kind=dp)*(ny*wp/ydirps%L)**2
       end do
       do i = ny/2 + 2, ny
-         yky(i) = yky(ny-i+2)
-         eys(i) = eys(ny-i+2)
-         yk2(i) = yk2(ny-i+2)
+        yky(i) = yky(ny-i+2)
```

Comment:
Looks like findent (at least with current settings) doesn't enforce anything between operators. (I think this particular line is not compatible with our current preference (spacing around `-` and `+` but no spacing around `*` and `/`) it would be nice if findent can identify and enforce a spacing based on our preference.

Reply:
findent doesn't support modifying these things. It is really mainly doing indentation.

Reply:
Its actually good. Because fprettify is too restrictive with spacing around operators, and I don't really like it. For example, when passing a keyword argument to a function, I would rather not using spaces around `=`, but of course prefer spaces around assingment `=` normally. And fprettify can't tell them apart and want you to pick one and use it everywhere. Also with `if` and `allocate`. You may want space after `if`, but no space after `allocate`, but fprettify won't let you do this because it considers these two in the same group and the rule is defined for the whole group...